The Dropper Community Edition/map.xml

```diff
@@ -105,24 +108,25 @@
     <point id="stage-15">205.5,252,102.5</point>
     <!-- Portals/score boxes at end of stage -->
     <union id="stage-ends">
-        <cuboid id="end-stage-1" min="25,2,-1" max="28,6,3"/>
-        <cuboid id="end-stage-2" min="29,7,1" max="27,11,-2"/>
-        <cuboid id="end-stage-3" min="133,2,-1" max="136,6,3"/>
-        <cuboid id="end-stage-4" min="187,2,-1" max="190,6,3"/>
-        <cuboid id="end-stage-5" min="241,2,-1" max="244,6,3"/>
-        <cuboid id="end-stage-6" min="25,2,50" max="28,6,53"/>
-        <cuboid id="end-stage-7" min="79,2,50" max="82,6,53"/>
-        <cuboid id="end-stage-8" min="133,2,50" max="136,6,53"/>
-        <cuboid id="end-stage-9" min="187,2,50" max="190,6,53"/>
-        <cuboid id="end-stage-10" min="241,2,50" max="244,6,53"/>
-        <cuboid id="end-stage-11" min="25,2,101" max="28,6,104"/>
-        <cuboid id="end-stage-12" min="79,2,101" max="82,6,104"/>
-        <cuboid id="end-stage-13" min="133,2,101" max="136,6,104"/>
-        <cuboid id="end-stage-14" min="187,2,101" max="190,6,104"/>
-        <cuboid id="end-stage-pre15" min="210,2,111" max="207,6,114"/>
-        <cuboid id="end-stage-15" min="241,2,101" max="244,6,104"/>
+    <cuboid id="end-stage-1" min="25,2,-1" max="28,6,3"/>
+    <cuboid id="end-stage-2" min="29,7,1" max="27,11,-2"/>
+    <cuboid id="end-stage-3" min="133,2,-1" max="136,6,3"/>
+    <cuboid id="end-stage-4" min="187,2,-1" max="190,6,3"/>
+    <cuboid id="end-stage-5" min="241,2,-1" max="244,6,3"/>
+    <cuboid id="end-stage-6" min="25,2,50" max="28,6,53"/>
+    <cuboid id="end-stage-7" min="79,2,50" max="82,6,53"/>
+    <cuboid id="end-stage-8" min="133,2,50" max="136,6,53"/>
+    <cuboid id="end-stage-9" min="187,2,50" max="190,6,53"/>

```

Comment:
`<cuboid/>` tags need to be indented under `<union>` properly.

Reply:
.. why remove permanent invisibility?